ratis-test/src/test/java/org/apache/ratis/RaftLogTruncateTests.java

```diff
@@ -178,9 +179,11 @@ void runTestLogTruncate(MiniRaftCluster cluster, RaftServer.Division oldLeader,
       LOG.info("done");
     }
 
-    // kill the old leader
+    // kill remaining peers
     LOG.info("Before killServer {}: {}", oldLeader.getId(), cluster.printServers());
```

Comment:
Match `After killServer` message?

```suggestion
    LOG.info("Before killServer {}: {}", remainingPeers, cluster.printServers());
```